cmd/dinghy.go

```diff
@@ -147,52 +155,52 @@ func Setup() (*logr.Logger, *web.WebAPI) {
 		migration.Execute()
 		migration.Finalize()
 
-	} else if config.SQL.Enabled  && config.SQL.EventLogsOnly {
+	} else if config.SQL.Enabled && config.SQL.EventLogsOnly {
 		// Hybrid SQL mode just for eventlogs
 		sqlClient, sqlerr := database.NewMySQLClient(&database.SQLConfig{
 			DbUrl:    config.SQL.BaseUrl,
 			User:     config.SQL.User,
 			Password: config.SQL.Password,
-			DbName:	  config.SQL.DatabaseName,
+			DbName:   config.SQL.DatabaseName,
 		}, log, ctx, stop)
 
 		if sqlerr != nil {
 			log.Fatalf("SQL Server at %s could not be contacted: %v", config.SQL.BaseUrl, err)
 		}
 
-		redisClient := cache.NewRedisCache(newRedisOptions(config.Redis), log, ctx, stop, true)
+		redisClient := cache.NewRedisCache(newRedisOptions(config.SpinnakerSupplied.Redis), log, ctx, stop, true)
 		if _, err := redisClient.Client.Ping().Result(); err != nil {
-			log.Fatalf("Redis Server at %s could not be contacted: %v", config.Redis.BaseURL, err)
+			log.Fatalf("Redis Server at %s could not be contacted: %v", config.SpinnakerSupplied.Redis.BaseURL, err)
 		}
 
 		redisClientReadOnly := cache.RedisCacheReadOnly{
 			Client: redisClient.Client,
 			Logger: redisClient.Logger,
 		}
 
-		logEventsClient = &(logevents.LogEventSQLClient{ SQLClient: sqlClient, MinutesTTL: config.LogEventTTLMinutes })
+		logEventsClient = &(logevents.LogEventSQLClient{SQLClient: sqlClient, MinutesTTL: config.LogEventTTLMinutes})
 		persitenceManager = redisClient
 		persitenceManagerReadOnly = &redisClientReadOnly
 
 	} else {
 		// Redis mode
-		redisClient := cache.NewRedisCache(newRedisOptions(config.Redis), log, ctx, stop, true)
+		redisClient := cache.NewRedisCache(newRedisOptions(config.SpinnakerSupplied.Redis), log, ctx, stop, true)
 		if _, err := redisClient.Client.Ping().Result(); err != nil {
-			log.Fatalf("Redis Server at %s could not be contacted: %v", config.Redis.BaseURL, err)
+			log.Errorf("Redis Server at %s could not be contacted: %v", config.SpinnakerSupplied.Redis.BaseURL, err)
```

Comment:
Why remove the Fatal here? My understanding of the branching now is that we'll revert to Redis if SQL isn't available, which would imply that persistence won't work correctly. Is that intended?

Reply:
That’s a good point, I just updated it.

That was one of the first changes I did, it made sense at that moment haha.